db/migrate/20170829134918_fix_default_value.rb

```diff
@@ -0,0 +1,5 @@
+class FixDefaultValue < ActiveRecord::Migration
+  def change
+    change_column :users, :phone, :decimal, :default => 0
```

Comment:
'0' is a value and the default phone number cannot be set.
A phone number is not a decimal.